### PR TITLE
Backup banner fix

### PR DESCRIPTION
**Pull request by Rockindash**
Backup banner logic fixed for Reshare, Issue #879

### CODE REVIEW

File: VultisigApp/VultisigApp/Views/Keygen/KeygenView.swift [hunk 1]
```diff
@@ -198,6 +206,15 @@ struct KeygenView: View {
             oldResharePrefix: oldResharePrefix
         )
     }
+    
+    private func setDoneData() {
+        if tssType == .Reshare {
+            vault.isBackedUp = true
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/Views/Keygen/KeygenView.swift:212]: I think this is a misunderstanding , after reshare , we actually want user to backup their vault

Comment (johnnyluo) [reply VultisigApp/VultisigApp/Views/Keygen/KeygenView.swift:212]: set `vault.isBackupUp = false`

File: VultisigApp/VultisigApp/Views/Keygen/KeygenView.swift [hunk 2]
```diff
@@ -38,12 +38,11 @@ struct KeygenView: View {
         }
         .navigationBarBackButtonHidden()
         .navigationDestination(isPresented: $viewModel.isLinkActive) {
-            BackupVaultNowView(vault: vault)
+            destination
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/Views/Keygen/KeygenView.swift:41]: revert this change

File: VultisigApp/VultisigApp/Views/Keygen/KeygenView.swift (hunk 1 above)
Comment (realpaaao) [reply VultisigApp/VultisigApp/Views/Keygen/KeygenView.swift:212]: Yes, after import = no banner.
after reshare = banner

**Review by johnnyluo — APPROVED** [review]
LGTM

**Review by enriquesouza — APPROVED** [review]
LGTM
